packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -46,18 +46,32 @@ abstract class Layer extends AbstractNode with DiagnosticableTreeMixin {
   // Whether this layer has any changes since its last call to [addToScene].
   //
   // Initialized to true as a new layer has never called [addToScene].
+  //
+  // The final value is only valid after calling [updateSubtreeNeedsAddToScene].
```

Comment:
not sure what "the final value" means here

Reply:
Done. Expanded the docs.